### ISSUE TITLE
Bug: Cache warmup schlägt fehl

### ISSUE THREAD

**Issue by datengraben** [wielebenwir/commonsbooking#1636]
Ein statischer Aufruf einer nicht-statischen Methode ```execute``` von ```MapShortcode``` führt in der 2.9.4 Version in unserer Instanz zu einer Exception und der Aufruf bricht ab.

Auswirkung ist denke ich gering, das Caching subsequenter normaler Seiten-Aufrufe greift ja auch ohne erfolgreichen warmup-Aufruf.

Folgender Fehler: ```Uncaught error: non-static method commonsbooking\map\baseshortcode::execute() cannot be called statically in /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-content/plugins/commonsbooking/src/service/cache.php:367```

Vollständiger Trace:

```
Uncaught error: non-static method commonsbooking\map\baseshortcode::execute() cannot be called statically in /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-content/plugins/commonsbooking/src/service/cache.php:367 
stack trace: 
#0 /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-content/plugins/commonsbooking/src/service/cache.php(263): commonsbooking\plugin::runshortcodecalls(array) 
#1 /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-includes/class-wp-hook.php(324): commonsbooking\plugin::warmupcache('') 
#2 /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-includes/class-wp-hook.php(348): wp_hook->apply_filters('', array) 
#3 /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-includes/plugin.php(517): wp_hook->do_action(array) 
#4 /home/webpages/lima-city/dasallrad/wordpress_de-2023-05-23-9d15cd/wp-admin/admin-ajax.php(192): do_action('wp_ajax_cb_cach...') 
#5 {main} thrown
```

Hier die betreffende Zeile:

https://github.com/wielebenwir/commonsbooking/blob/e7636a305f0c9c5cfb5978385dab088aeb503314/src/Service/Cache.php#L433
